core/tests/performance_framework/perf_services.py

```diff
@@ -240,8 +240,11 @@ def _setup_driver(self, proxy=None, use_proxy=False):
             chrome_options = webdriver.ChromeOptions()
             # Disable several subsystems which run network requests in the
             # background. This helps reduce noise when measuring network
-            # performance.
+            # performance. Also, disable prerendering by chrome which renders
+            # a page in the background leading to wrong test results.
```

Comment:
Turning on prerendering will negatively impact pages (note: only the page size metric and no of requests will be affected and that too because we are summing up everything sent on the wire) that have it and it will have no impact when testing pages that are being prerendered. That said, prerender has a lower priority than the current page resources and the impact should not be very determinable.